.release/security-scan.hcl

```diff
@@ -2,20 +2,25 @@
 # SPDX-License-Identifier: BUSL-1.1
```

Comment:
Run `hclfmt` on these HCL files please.

Reply:
Should be done, https://github.com/hashicorp/nomad/pull/20465/commits/dfff95becaf6f28364e2203cd83ae0bb8574f997